orchestration/OrchestrationService/ModuleConfigurationDeployment.ps1

```diff
@@ -628,12 +655,51 @@ Function Get-SubscriptionInformation {
     }
 }
 
+Function Get-CacheStorageInformation {
+    [CmdletBinding()]
+    param (
+        [Parameter(Mandatory=$true)]
+        [hashtable] 
+        $ToolkitConfigurationJson
+    )
+
+    try {
+        $cacheStorageInformation = @{
+            StorageType = ''
+        };
+
+        if ($ToolkitConfigurationJson.Configuration.Cache -and
```

Comment:
In this logic we should default to local storage type if there is no file or no Configuration.Cache object,

Reply:
Updated.